storage/db-tool/src/backup_maintenance.rs

```diff
@@ -45,13 +49,19 @@ impl Command {
     pub async fn run(self) -> Result<()> {
         match self {
             Command::Compact(opt) => {
-                let compactor = BackupCompactor::new(
+                ensure!(
+                    opt.expiration_duration > MIN_COMPACTION_EXPIRATION_SECS,
+                    "Expiration duration must be greater than {} seconds",
+                    MIN_COMPACTION_EXPIRATION_SECS
+                );
```

Comment:
Well we already have a default value, and help message indicating what's the risk of making it short, I feel we don't need to be too strict here -- we ourselves might want to do some yolo operations in production some day :P 

up to you (oh, but 12 hours seems really long for a hard limit)